environments/staging/staging.tfvars

```diff
@@ -0,0 +1 @@
+env = "aat"
```

Comment:
why does this need setting now and not before? (as far as I can tell)

Reply:
This is the way it is in [master](https://github.com/hmcts/azure-private-dns/blob/1e7b4119cbea3206ddd327a723850f6a4fa7160c/components/staging/variables.tf#L7). Just re-adding it back.  Another alternative will be to add `staging` in common tag module [on this line](https://github.com/hmcts/terraform-module-common-tags/blob/093dcb24e2d1973945b093dc31506222fda0c9c0/main.tf#L45)

Reply:
or rename the folder to be stg